setup_fleet_server.sh

```
@@ -12,6 +12,7 @@ set -e
 SCRIPT_DIR="$(dirname "$(realpath "$0")")"
 
 # --- Variables ---
+ELK_VERSION="9.4.4"
```

Shellcheck (0.11.0)

[warning] ELK_VERSION appears unused. Verify use (or export if used externally).

(SC2034)